extensions/src/main/java/org/pentaho/platform/web/http/api/resources/WorkerNodeActionInvokerAuditor.java

```diff
@@ -70,4 +70,9 @@ protected void makeAuditRecord( final float time, final String messageType,
   private String getValue( Map<String, Serializable> actionParams, String key ) {
     return actionParams.get( key ) != null ? actionParams.get( key ).toString() : "";
   }
+
+  @Override
+  public boolean isSupportedAction( IAction action ) {
+    return false; // not needed
```

Comment:
Don't see why not. 

This can be done as a follow-up "cleanup" process. As it stands now, it is not a blocker. I'd request that someone with a better understanding of the intents of this Auditor decorator class can do such a change.